manager-dashboard/app/views/NewProject/index.tsx

```diff
@@ -326,6 +330,30 @@ function NewProject(props: Props) {
                 }
 
                 const database = getDatabase();
+                const projectTopicKey = projectTopic?.toLowerCase() as string;
```

Comment:
We might need to update this logic to trim whitespace and also replace multiple whitespaces to a single space.